data/mods/Xedra_Evolved/snippets/bloodthorne_note_snippets.json

```
@@ -0,0 +1,18 @@
+[
+  {
+    "type": "snippet",
+    "category": "note_bloodthorne_druid",
+    "text": [
+      {
+        "id": "xe_bloodthorn_note_01",
+        "name": "Meditations",
+        "text": "The elders want me to write down what I feel during weekly meditations but I don't see the point.  I've never been good at concentrating.  I go out into the forest and I sit and pretty soon there's some bugs crawling on or my back itches or I get a cramp on my leg and it breaks my concentration.  I've gotten pretty used to the heartvines moving, that's not the problem, it's the *other* stuff I can't get past.\n\nIt's weird, right?  You'd think that having a plant in my blood would be all I can think about but I got used to it in a week.  ̶A̶n̶y̶t̶h̶i̶n̶g̶ ̶i̶f̶ ̶I̶ ̶c̶a̶n̶  No, not yet.  I have to find the thing that did it, then I'll write it all down.  Now I just have to stop getting those cricks in my heck when I'm looking up through the leaves."
```

```suggestion
        "text": "The elders want me to write down what I feel during weekly meditations but I don't see the point.  I've never been good at concentrating.  I go out into the forest and I sit and pretty soon there's some bugs crawling on me or my back itches or I get a cramp in my leg and it breaks my concentration.  I've gotten pretty used to the heartvines moving, that's not the problem, it's the *other* stuff I can't get past.\n\nIt's weird, right?  You'd think that having a plant in my blood would be all I can think about but I got used to it in a week.  ̶A̶n̶y̶t̶h̶i̶n̶g̶ ̶i̶f̶ ̶I̶ ̶c̶a̶n̶  No, not yet.  I have to find the thing that did it, then I'll write it all down.  Now I just have to stop getting those cricks in my neck when I'm looking up through the leaves."
```

Since this is handwritten did you want the mispellings and missing words?

---

No, those mispellings are mine, not the bloodthorne initiate's